### PR TITLE
Shipit end to end aurora example

### PR DESCRIPTION
**Pull request by wyrzyk**

### CODE REVIEW

**Comment by afaruga-atlassian** [comment]
Nice, TBH I have kinda similar idea, [shipit ideas
](https://hello.atlassian.net/wiki/spaces/~662044858/pages/1066572640/ShipIt+ideas)
```
MultiReplicaIntegrationTests
AuroraExperimentalLab - CI
```

I I wanted to achieve that in Jira CI and have a way to run connpool-db-replica integration-tests on provisioned aurora.

I was considering  [Aurora+Serverless+v1](https://hello.atlassian.net/wiki/spaces/MICROS/pages/1209140513/Early+Access+Aurora+Serverless+v1+PostgreSQL+Cloudformation)
How do you provision Aurora? Do you have an idea to maintain it for longer period of time?

**Comment by wyrzyk** [comment]
>  How do you provision Aurora? Do you have an idea to maintain it for longer period of time?

I planned to use the ShipIT AWS account but was blocked by permission issues. I'm using my personal AWS account for development now. For the long term, we'd need a publicly available aurora test/dev cluster. I think it shouldn't be a problem as the cost is low and we'll not going to keep customer's data on the instance.

File: src/test/java/com/atlassian/db/replica/it/example/aurora/replica/api/SynchronousWriteConsistency.java [hunk 1]
```diff
@@ -0,0 +1,138 @@
...
+import static java.lang.Math.min;
+
+/**
+ * Waits until writes propagate to replica.
+ * It doesn't depend on a cross-JVM cache.
+ */
+
+// TODO add also DeferredSynchronousWriteConsistency implementation
+// TODO multi replica support
+public class SynchronousWriteConsistency implements ReplicaConsistency {
```
Comment (wyrzyk) [inline src/test/java/com/atlassian/db/replica/it/example/aurora/replica/api/SynchronousWriteConsistency.java:23]: Deferred WAW is out of the scope for the initial MultiReplica support in the lib.

**Comment by wyrzyk** [comment]
Moved to https://github.com/atlassian-labs/db-replica/pull/106

**Pull request closed without merging**